community/mei-board.md

```diff
@@ -22,14 +28,10 @@ The MEI Board manages the property and business of MEI. It promotes the developm
 * **Laurent Pugin**, RISM Digital Center, Bern
 * **Kristina Richts-Matthaei**, Musikwissenschaftliches Seminar Detmold/Paderborn
 
-### Term 2020–2022
-
-* **Johannes Kepper**, Musikwissenschaftliches Seminar Detmold/Paderborn
-* **Perry Roland**, University of Virginia Library, Charlottesville (previous terms 2015–2018)
-* **Martha Thomae**, McGill University, Montréal
-
 ### Previous Board Members
 
+* **Martha Thomae**, McGill University, Montréal (2020–2022)
+* **Perry Roland**, University of Virginia Library, Charlottesville (2015–2018, 2020–2022)
```

Comment:
If I remember correctly, we used alphabetical order for previous Board members. @bwbohl Can you please sort accordingly?